hrm-service/src/models/case.js

```diff
@@ -1,3 +1,5 @@
+// Can we delete this file?
+
```

Comment:
Ah I remember now, @stephenhand this is still used by the permissions framework so no, it can't be deleted yet.  We could instead than using the `Cases` model as exported by Sequelize, use the `Case` type instead (if `cancan` does not works on types we'll need to move that information to a class tho).

Reply:
Either that or we could just use a stub class purely to hang permissions off? I don't think the class cancan uses actually needs to do anything if I understand the way we use cancan correctly?